apps/test/unit/templates/rubrics/RubricContainerTest.jsx

```diff
@@ -23,13 +23,29 @@ import teacherRubric from '@cdo/apps/templates/rubrics/teacherRubricRedux';
 import teacherSections, {
   setStudentsForCurrentSection,
 } from '@cdo/apps/templates/teacherDashboard/teacherSectionsRedux';
-import {
-  RubricAiEvaluationLimits,
-  RubricAiEvaluationStatus,
-  LevelStatus,
-} from '@cdo/generated-scripts/sharedConstants';
+import {RubricAiEvaluationLimits} from '@cdo/generated-scripts/sharedConstants';
 import i18n from '@cdo/locale';
 
+import {
+  stubFetch,
+  studentAlice,
+  levelNotTried,
+  levelSubmitted,
+  notAttemptedJson,
+  notAttemptedJsonAll,
+  readyJson,
+  readyJsonAll,
+  pendingJson,
+  runningJson,
+  successJson,
+  successJsonAll,
+  defaultRubric,
+  noEvals,
+  oneEval,
+  defaultStudentInfo,
+  mockAiEvaluations,
+} from './rubricTestHelper';
```

Comment:
note to reviewers: the only changes to this file are to extract shareable content to rubricTestHelper.